classes/class-wc-connect-service-schemas-store.php

```diff
@@ -60,6 +65,17 @@ protected function update_last_fetch_timestamp() {
 			WC_Connect_Options::update_option( 'services_last_update', time() );
 		}
 
+		public function get_last_fetch_result_code() {
+			return WC_Connect_Options::get_option( 'services_last_result_code' );
+		}
```

Comment:
Apparently this function is not being used, should we remove it?

Reply:
no, I missed committing a file that uses it